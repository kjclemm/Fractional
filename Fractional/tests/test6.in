1000 30
1288 1927 1406
1938 1346 1371
2823 2842 1431
1903 2537 1387
72 1215 845
2249 2434 861
1972 304 1679
440 162 1050
1133 1220 2821
731 2008 308
19 2010 1675
2213 360 1832
2448 356 2003
2528 952 683
1484 2408 559
107 1382 2665
2837 2895 966
2263 2251 455
2033 1271 2321
2714 2152 1933
1770 769 2560
1223 1274 1518
2529 1034 1003
430 2241 1904
520 1574 2419
1892 1474 557
591 1436 133
136 418 347
242 1862 2122
2328 32 1823
1098 536 2948
1443 582 2802
2704 1699 2746
47 2903 498
1135 2629 2480
2871 1747 430
2474 564 1045
1281 2446 1761
120 2581 1322
2180 1975 1267
546 195 1332
204 114 756
294 1515 2250
1951 1428 381
1326 1528 2428
202 2197 2279
2398 1371 154
915 58 1317
2281 1874 2336
2994 843 2402
2991 1492 2862
2586 1226 2057
1267 2466 2547
377 2172 705
2109 690 2392
2798 1788 882
357 350 871
2404 1732 463
2004 2659 324
2486 2192 1005
711 1755 2468
12 1533 644
311 561 877
2446 1784 2764
2903 1543 595
1091 2893 1427
1001 2814 2508
1800 306 663
1214 2226 1448
455 1583 1913
539 652 736
2803 257 1806
694 120 623
2855 1601 570
1941 1510 2911
784 449 2805
151 571 896
2030 352 1949
1193 1563 28
273 379 507
1321 2180 2741
2179 1728 569
1606 225 2627
870 305 117
830 103 762
2744 798 2571
2317 2755 1305
2385 1825 2748
1415 797 255
816 438 110
1295 55 1394
2127 893 2498
185 2682 2587
2594 2769 2106
1582 846 817
1599 95 462
1838 1524 131
2106 398 1610
1861 1656 726
1250 1022 1212
2669 1945 477
1494 184 553
678 704 1470
2270 676 697
337 2420 1509
576 2480 604
2443 1530 815
1414 30 2967
1523 108 1782
42 268 2153
2636 2323 770
9 1459 1639
1182 188 1813
387 2304 1349
2739 129 2238
2549 249 1152
226 353 349
1645 1284 1369
2298 868 1075
1403 2360 579
1292 614 2051
1284 132 1429
1070 2523 2103
1154 382 513
1526 1024 1541
145 1958 1676
1319 107 1360
1866 1143 2611
1995 1566 856
2736 1756 1620
2043 727 1089
1411 546 245
2218 2025 2382
2959 332 315
2910 2247 2915
1208 2422 2980
2804 961 355
1940 170 1579
2432 59 291
1118 6 2009
1550 2765 706
2203 2514 2073
1348 131 283
1011 2869 2159
1388 1196 416
2833 1464 268
1840 2165 1042
2737 2088 330
1066 2343 1398
780 644 1438
2657 2331 2693
438 1343 716
489 212 688
2069 1982 1139
1538 890 1488
2254 2243 1690
2610 734 1638
752 2184 2773
352 2292 828
2305 768 1702
271 1685 1546
2010 1613 2312
2929 208 272
1268 900 314
839 336 2267
2546 2756 950
641 2222 2447
278 1010 1707
1130 1448 282
419 543 2582
788 2610 2252
2673 1404 2522
1920 2681 2942
461 1833 2888
332 2886 359
2891 2392 2593
2539 1083 2984
229 625 538
660 1750 1379
2174 2580 1985
1212 1953 2109
536 362 2577
2271 1611 2660
2175 2433 674
2872 2217 1230
48 688 1838
1666 2892 2846
2242 1597 806
1617 2064 661
642 779 2206
2670 2997 383
2014 532 1029
2841 2939 2525
1127 405 1199
702 409 1216
789 2369 2829
904 1430 1554
1475 227 2523
2159 1527 2733
290 1156 1422
2821 1288 1074
1278 2200 1337
2488 1630 1423
2288 2368 2341
2817 1565 2985
1126 730 2370
2789 1072 1467
2021 1040 684
1121 400 1835
2300 2372 2126
848 842 590
434 355 1529
2703 2404 103
842 1903 637
1007 1364 413
1784 1203 276
1353 2141 371
2423 423 33
1100 2018 424
2521 2587 605
722 2325 2852
645 1439 2248
483 2938 2751
347 2689 2297
2522 831 1125
1850 1693 825
1497 1709 2451
1164 587 2516
1707 2952 128
1198 372 2164
1008 1867 1220
1352 1652 2094
1396 619 144
1247 2971 763
155 440 2968
2283 90 87
407 575 2084
2316 932 19
2274 87 2504
840 2535 2414
2387 1864 2176
2299 302 673
2237 1635 2075
506 1057 2227
1501 47 1863
1140 2925 293
446 1525 2135
1815 634 1040
1571 2407 2439
2507 632 741
2684 1185 2143
907 50 2594
696 2568 1773
44 528 2338
1225 479 1210
871 1416 2553
2579 1270 2616
768 2822 250
2713 2779 1390
171 1870 1703
325 2989 228
491 2099 2858
782 884 218
1581 283 331
1297 430 2963
2877 1145 328
2135 434 1158
166 889 1436
220 1582 1242
622 1889 1762
502 1378 1841
584 540 2406
2050 1238 145
2957 913 1101
793 1159 524
2214 2506 1894
1304 1324 94
1277 2319 748
1200 1075 2478
2816 2991 1487
2519 888 1989
1997 2841 976
1580 1531 1375
2119 1904 580
1113 70 2783
51 2457 1170
2768 1816 1667
2963 134 2413
1913 399 2837
497 2385 2549
963 1727 685
2367 2326 2904
443 2341 840
1379 1199 982
1300 2468 2931
1695 1399 656
2919 2780 2142
1361 2473 2390
1906 897 2188
2809 2906 378
680 1809 2711
2123 1720 1352
1203 2738 922
2616 265 2623
2416 1713 2801
1910 143 1986
1255 1512 938
1514 2083 2818
1915 985 2048
318 1206 2633
630 2930 1400
1919 2144 583
431 2285 1919
1560 2716 240
897 1511 2847
61 2960 2955
2878 2607 887
1244 1167 385
2653 2359 2011
1593 1188 223
2016 2100 625
770 1165 1115
1767 2538 1649
2430 181 190
428 992 1881
932 1100 2254
2009 1886 2356
1628 2788 23
911 1980 480
1491 983 1738
52 2313 2659
1345 2594 1937
2308 1778 289
2794 2399 2816
279 760 1875
1480 2173 1938
2690 974 1514
713 1493 1757
1369 2005 1173
2442 354 207
2366 2735 932
2232 331 2344
2255 2269 1538
1727 2793 2505
2139 1898 501
206 1409 2959
967 1522 953
2560 1518 502
847 1554 2854
433 2307 1745
2935 2030 545
2087 2436 1584
2677 2472 2977
1064 1454 2589
2158 380 791
286 2936 2919
411 1740 1194
2774 435 2889
460 2700 2054
1914 324 2961
2591 506 439
682 2932 2740
634 2657 2456
1230 1860 584
2998 2522 2486
2854 2486 2410
510 1564 84
1662 1153 485
2517 1328 2372
760 2260 264
1823 377 2138
342 2803 1053
1527 2162 1825
652 2985 881
2695 2771 527
2810 1252 2763
71 621 956
2084 964 996
1856 2966 868
2357 389 2067
1852 1696 1739
391 578 1313
2773 1258 352
666 2403 2918
2291 1792 2831
1597 2295 151
6 2571 1891
2542 2835 1107
1446 361 744
1725 2563 187
927 2482 100
749 2484 2389
2154 2469 2897
2767 2724 332
364 1626 449
2754 251 312
369 2653 2561
379 1243 1818
2735 2397 1577
974 845 2870
1773 2950 2814
2088 1047 1226
2573 1660 2037
735 620 608
2572 2856 1452
1464 1177 875
1485 1723 2970
2526 1 129
1642 1419 2327
1192 795 2873
1269 2255 2875
1705 678 1683
1144 1503 231
1509 2423 2964
191 1184 2381
1459 2688 2242
118 2219 2197
823 1169 121
1049 2236 1201
1690 1079 1254
2090 1309 444
2210 552 1828
2916 1393 2407
941 2082 2573
863 2648 2607
1438 1542 883
2961 1853 2536
2257 1521 2603
2273 1276 86
562 291 1346
2721 1798 589
2341 2919 1359
101 2742 241
2633 1405 1181
441 2565 118
1452 451 294
615 147 2940
1829 1877 1583
620 918 914
525 975 2992
50 1643 292
875 991 192
1559 1769 2018
2458 1568 2001
2209 1682 1471
1346 1012 150
230 724 345
2788 2805 1203
832 1559 405
2928 192 2260
484 203 1256
2489 339 398
1271 677 2203
2373 2759 1041
2968 1386 759
2024 2962 353
2502 189 1364
1602 2606 793
128 2022 34
213 2808 997
914 2719 1096
2161 333 2309
945 2072 1135
2374 2690 2033
2974 818 1335
806 2787 2160
1746 410 1183
404 2551 1593
2444 464 1590
453 2026 1217
1857 1389 991
1 1539 732
1739 15 2022
2262 776 1356
2914 720 2642
2212 1570 957
192 1615 2661
983 679 2753
583 327 1426
448 2143 988
1380 1986 1466
295 2630 196
746 1560 2425
1134 1751 2387
1253 1073 551
2550 296 440
312 1092 2060
2484 939 2
1827 492 205
1865 527 830
1658 2899 2349
1528 2749 2956
1471 2053 2195
761 1311 2973
2699 1552 535
172 942 1365
1591 2633 1416
856 2627 2035
1834 1964 1804
8 1855 2839
1667 531 1882
1240 1690 1388
1787 2429 978
2067 2984 1469
56 2762 2602
619 1469 1727
1786 995 1312
838 2979 2784
772 681 2626
2195 282 492
409 96 750
1359 142 609
781 2827 2697
667 1190 2933
1406 2986 675
1094 584 746
691 1948 167
1103 183 666
237 1151 515
211 2727 2158
54 1359 1831
1163 1850 2887
11 2157 2476
2867 261 1413
1419 1808 1417
2259 1375 1454
936 1281 357
2604 773 323
1712 604 1443
805 1987 1524
2555 2084 642
1567 1579 166
876 1585 346
1873 2076 170
2435 937 263
2654 494 305
261 912 402
2105 1137 1512
1056 57 2038
1635 214 561
1285 1549 2141
981 1062 870
2236 600 1424
1631 1653 702
2772 1133 1116
1778 766 766
1588 2454 662
2893 1846 2825
753 2555 2275
703 2596 2696
602 2626 1843
1837 1593 792
1444 1403 1513
867 560 2212
581 2087 217
1398 2079 341
1525 1294 2007
330 1801 2318
2527 1620 175
2718 544 747
359 2701 571
1263 25 1791
210 1725 232
1176 1911 852
74 2558 258
3000 1627 765
582 40 2820
2913 931 617
2438 1182 2020
1812 469 1036
2758 2142 1714
2827 2134 350
194 1180 2634
2290 2645 2934
154 541 214
2638 656 2085
2874 1445 2786
2162 392 406
1664 1614 758
1461 422 2125
1157 737 2302
2762 408 2418
1412 2261 2646
2120 807 948
2455 2221 2266
1884 935 41
1535 1129 2299
1309 1426 157
2852 1067 1059
2352 2562 1530
939 589 2187
367 1063 2289
1301 1481 2065
1407 635 1640
1624 2046 2532
1611 1668 1544
1547 613 1099
2719 2301 319
1206 1806 2165
603 2191 1182
2413 1059 2347
764 1878 1964
2864 2206 1489
499 383 2843
338 2496 1000
577 320 1945
1290 1095 1868
686 2508 1292
2932 1757 2464
2258 2320 1607
2079 1821 2201
432 2498 2824
2743 793 2669
625 2174 745
596 1780 2345
578 717 1651
2181 1943 1361
444 1749 1408
1985 1921 1412
1237 2774 820
726 2074 183
592 1741 2809
2476 1926 2261
1286 2843 321
1371 137 2284
1889 26 2690
2548 2128 634
1276 2057 2024
986 1774 659
754 1912 210
1600 2545 1608
609 1269 738
2711 1501 2291
39 554 1657
1002 1036 2455
2475 1387 2366
570 1631 1758
2839 639 61
2206 1936 740
492 2987 676
919 2055 1411
1897 1573 2894
586 2185 2734
529 2766 1901
2845 2699 2221
1874 2543 599
627 1369 450
2365 2231 2397
2512 1628 1781
1378 2381 2726
2880 33 2493
2129 1581 2792
412 1327 25
2612 576 2431
2456 1851 1920
637 1141 1447
392 412 2115
852 643 2823
1028 2097 1205
1968 1504 2383
1657 2954 1076
2244 819 2139
2640 1192 1902
181 2092 1765
2887 5 2271
2071 716 1899
909 272 921
1757 2557 720
1755 1058 2116
721 370 2575
2051 1598 48
2806 1221 1492
1863 1931 2196
1930 2043 2674
951 415 2510
528 505 795
662 631 1338
1483 2861 1068
604 783 525
996 1003 318
1179 641 2257
57 1094 699
2972 199 2689
268 1733 645
109 2413 879
2920 2601 1441
2950 1148 2411
1692 229 1750
896 923 262
55 1829 1711
1181 2810 1178
1521 838 473
2777 837 2452
2984 647 1971
1205 1263 1752
2487 2071 2656
2662 1087 1897
2231 524 1318
1186 854 1083
2509 2902 576
2312 1013 2614
1454 526 903
935 2672 944
644 1968 1463
1306 844 629
2996 588 2735
2801 2526 2263
1386 2717 2863
1055 2120 382
91 110 1958
139 2988 568
170 21 603
2370 1259 737
1966 1678 2255
121 1261 2473
1974 549 1870
122 1244 1731
542 2234 1002
16 1973 940
376 1771 2909
610 860 393
123 1993 2241
2199 2432 984
463 2977 1419
1502 1278 2185
869 2980 971
1590 523 1249
1328 2112 1996
607 1427 1308
159 2844 1605
533 593 2855
776 1056 1780
2376 2850 592
734 1197 1276
1694 411 2215
708 740 1847
2892 2515 862
1190 926 1622
200 1743 388
1069 220 1699
894 2211 342
390 1892 1888
559 2488 1480
1442 542 30
34 2909 2169
2723 802 2788
2847 207 2316
2447 1366 963
2565 1541 1376
1141 1591 774
623 2995 1319
2461 966 2102
1500 816 2850
1551 135 811
2697 2062 2534
2384 1783 1401
351 1914 320
2269 1260 410
1062 876 235
742 2151 1016
938 567 755
844 2329 1801
1762 359 1864
143 2136 2518
2717 1166 1157
2589 1553 1982
2238 1101 854
2351 2665 2313
87 1066 2807
79 929 1553
439 247 178
984 2675 1978
2536 792 930
2606 2015 147
80 829 853
2320 2912 2712
1820 2229 2346
1647 2405 248
2334 41 1231
305 2775 2939
890 1781 2039
1496 2870 1571
2307 1575 1946
2296 1179 565
2354 1902 619
1260 1827 743
2879 2036 722
845 223 833
1536 1176 172
2297 1885 436
1665 270 201
13 1111 894
445 93 230
1231 2098 1955
1515 2612 91
2970 2410 1382
2580 442 2367
1078 1729 2342
96 2298 51
2632 1377 362
240 1983 2225
2600 1805 2524
2198 2006 2363
1975 2363 908
1356 880 796
2465 2156 1063
1138 1994 2246
1901 1967 2590
481 2996 69
2980 2406 2531
1797 2804 2140
926 1264 1645
903 2577 66
495 1802 83
2941 2802 602
1023 1544 574
246 85 958
2934 453 514
665 1019 2544
987 2160 2718
2338 2093 1977
1145 1629 204
2345 2101 2273
2368 583 1697
2927 1209 2869
526 1977 2667
372 2336 2016
418 139 426
2189 1225 2415
1722 2855 1240
263 2918 1051
1327 836 2838
2530 2764 1208
2930 11 1244
2358 1099 2357
1759 1390 2145
2533 1245 1451
1170 158 1850
349 2800 1730
2814 516 1837
1832 2459 855
687 468 16
962 1014 2236
564 1497 2630
288 763 572
2083 148 2996
1404 915 1486
1831 100 2580
651 1300 1344
464 2058 1056
1243 1026 2662
2457 1447 1594
679 1625 2470
2377 1858 445
658 2881 986
2219 2095 474
98 1859 1062
1783 274 1204
1795 615 876
4 1594 2396
2664 2118 81
2060 657 1044
7 472 1494
1839 448 2004
327 595 2032
654 1676 1481
866 987 1077
2979 2868 1556
747 2863 677
2239 902 1370
917 1122 1551
1227 439 1974
1508 1152 2637
2204 1317 1283
2639 1275 729
447 2560 649
834 825 438
514 2651 2148
2080 1487 1298
1128 692 1340
2034 2193 1381
189 976 1026
1226 2542 1316
2595 2875 300
1347 1679 560
701 2924 35
1153 1920 1052
2289 685 794
1229 873 354
2107 853 890
1736 1538 2971
1807 1776 536
2247 495 1294
826 1090 2507
401 1394 2905
912 1468 2899
2752 594 818
942 981 2923
850 2549 786
284 387 687
1934 2358 2813
2607 2531 1907
272 664 2880
1592 2289 838
2386 301 627
2401 373 1632
184 879 500
948 2649 954
292 1742 2245
378 2532 2287
1167 2056 2249
727 2481 1176
2944 1412 936
1678 2456 2061
626 1712 2647
733 2674 2409
672 233 142
1870 796 1054
1851 481 2562
2076 2852 1147
715 1580 2417
2516 882 1179
2708 1684 2044
1422 2552 2709
2339 1823 2705
1821 390 1686
2746 1353 2081
2287 309 351
2243 27 1232
2070 628 20
1177 381 2002
217 2342 1262
216 870 2670
2819 1212 149
751 2736 2388
2790 1191 2172
1738 2953 2319
712 1401 1117
1982 861 2132
134 102 254
35 2032 2618
1876 1779 2162
25 2848 848
548 378 1081
2905 159 2663
977 770 2120
2624 1599 620
1708 1021 490
1983 2643 2311
2152 857 563
2147 2718 648
1082 1868 1998
2740 726 1790
1622 1283 831
85 1214 2202
995 2293 1917
889 1837 2761
1693 533 226
383 1499 1887
1368 1280 632
1169 940 2588
916 488 57
2046 201 1461
643 2493 2975
2924 1268 470
1016 2729 1405
94 753 2471
2759 490 2728
245 722 2019
705 118 44
420 1142 858
663 1046 638
396 1596 1253
2007 414 1281
1594 73 2760
2786 441 2264
1220 895 1606
1641 691 65
70 1085 529
846 1952 441
1417 2294 2892
174 1550 1741
479 996 2936
873 1657 1437
737 2333 2481
1279 2447 2027
778 221 1146
998 2230 1536
2710 67 2191
585 2124 236
1324 1686 939
2073 244 2881
2688 1724 2229
2655 1107 1507
1959 74 739
1194 2578 184
1065 24 303
624 2040 2958
2115 826 2226
633 275 614
150 903 1184 3297 618 6479
553 2218 1621 7023 973 1930
973 2745 707 1580 754 2379
399 3013 1204 5311 711 2119
808 2418 1833 4831 1336 4348
403 4125 1921 3279 94 3485
490 4293 62 3934 1990 2808
594 3322 1462 5096 1617 5958
1813 2827 695 4277 489 5918
1559 6190 824 6263 1433 2962
852 5238 1128 2909 646 2091
1856 7118 261 2272 1738 4956
1052 4067 1407 3188 1450 2783
1459 3064 1365 6936 322 2144
683 2772 1659 4499 1994 3391
62 3694 32 3797 1633 2487
1923 3428 1217 6295 1054 4423
666 693 1389 4273 1909 3414
768 2899 416 3581 95 4712
1117 1171 658 1064 1413 4119
1055 1613 889 3006 583 2163
217 4302 1129 5181 1228 3348
427 6427 1929 5634 1781 7295
279 2392 1707 3827 620 4102
681 5096 36 2073 1142 6886
1291 5626 1510 7345 1419 3317
1377 4892 113 424 1959 2725
664 4012 929 3042 926 6383
1910 3797 1157 2468 988 3530
827 1033 1577 7370 465 3337
