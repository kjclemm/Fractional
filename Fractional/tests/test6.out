141
68
92
220
104
288
230
184
215
151
181
111
127
159
108
261
129
4
570
3
64
335
121
251
311
134
18
345
104
29
